gravitee-apim-e2e/api-test/applications/application-membership.spec.ts

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { afterAll, beforeAll, describe, expect } from '@jest/globals';
+import faker from '@faker-js/faker';
+import { ApplicationsApi } from '@management-apis/ApplicationsApi';
+import { forManagementAsAdminUser, forManagementAsAppUser, forPortalAsApiUser, forPortalAsSimpleUser } from '@client-conf/*';
+import { forbidden, succeed } from '../../lib/jest-utils';
+import { ConfigurationApi } from '@management-apis/ConfigurationApi';
+import { UsersApi } from '@management-apis/UsersApi';
+import { RoleScope } from '@management-models/RoleScope';
+import { ApplicationApi } from '@portal-apis/ApplicationApi';
+
+const applicationManagementApiAsAppUser = new ApplicationsApi(forManagementAsAppUser());
+const configurationManagementApiAsAdminUser = new ConfigurationApi(forManagementAsAdminUser());
+const usersManagementApiAsAdminUser = new UsersApi(forManagementAsAdminUser());
+const applicationPortalApiAsApiUser = new ApplicationApi(forPortalAsApiUser());
+const applicationPortalApiAsSimpleUser = new ApplicationApi(forPortalAsSimpleUser());
+const orgId = 'DEFAULT';
+const envId = 'DEFAULT';
+
+let application;
+let group;
+let userMember;
```

Comment:
it's not an issue you can let it like that but I would have add the types `ApplicationEntity`, `GroupEntiy` and `SearchableUser`